tests/bluetooth/tester/default.conf

```diff
@@ -1,6 +1,8 @@
 CONFIG_TEST=y
 CONFIG_UART_PIPE=y
 CONFIG_CONSOLE_HANDLER=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
```

Comment:
Hi,

As suggested above I have increased the system work queue stack size to "8192"
as per https://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE.html

But still no luck, I am able to see

" bt_driver: discarding event 0x3e"

Please suggest me the possible fix for the above issue, as its blocking our work.

Thank you.